### ISSUE TITLE
About feature

### ISSUE THREAD

**Issue by huizhang1032** [muhanzhang/pytorch_DGCNN#39]
How to output the feature of the last layer of the neural network of each graph  in the list of graphs to be tested ?

**Comment by muhanzhang** [comment]
You can add a "-extract_features" to your command to save the test graphs' embeddings to "extracted_features_test.txt". See this [line](https://github.com/muhanzhang/pytorch_DGCNN/blob/50f504131ca66382f3f078ae98e59aa3ae35b795/main.py#L235).

**Comment by huizhang1032** [comment]
OK.Thank you!

**Comment by huizhang1032** [comment]
If I input 100 graph objects to the model, the output feature is 100*32. Does 1*32 here represent the embed of a graph object?

**Comment by muhanzhang** [comment]
Hi, if you input 100 graphs, and set the final graph embedding size to 32, then you will get 100 * (1 + 32) dimensional matrix in your output txt. The 1 additional dimension corresponds to the predicted labels of all input graphs.

**Comment by huizhang1032** [comment]
I got it.Thank you very much！
